### ISSUE TITLE
Screen reader - integrate field name into aria-label

### ISSUE THREAD

**Issue by microbit-robert** [RaspberryPiFoundation/blockly#9423]
### Check for duplicates

- [x] I have searched for similar issues before opening a new one.

### Problem

For full block fields, the screen reader output doesn't include the field name. This is similar to focusing on a text input that doesn't have a name attribute or label. It would be really beneficial to have this added context for users.

### Request

Is it possible to include the current field name in the aria-label for full block and other fields so we can construct [value, field name, other role info]? The field name is often in the DOM as the aria-label for a sibling element to the one that takes focus. This may share some detail with https://github.com/google/blockly/issues/9420.

The current readout for the block below is "micro, input block, button". It would be more informative if it was possible to include the current field name so we get construct [value, field name, role info]. In this case, it would be "micro, TEXT, input block, button". There are more complex cases in MakeCode where the field name would be even more valuable.

<img width="315" height="129" alt="Image" src="https://github.com/user-attachments/assets/4d0a9789-a469-4044-bd2c-58d8918dc474" />

As these field names are not typically human-friendly, it would be great if there was a way to provide text that could be used in place of the default field name to improve this readout.

### Additional context

It would be good if we were able to user test with something like this in early November, but appreciate that the timeline is tight.

**Comment by microbit-robert** [comment]
This appears to be partially addressed by https://github.com/google/blockly/pull/9424. We now get the field name for some inputs such as text inputs and not others like dropdowns. One outstanding question about the order in the readout. I think for most normal web inputs / select elements, the value is read first, followed by the input name or label, should the same happen in Blockly?
